.changeset/tame-rings-study.md

```diff
@@ -0,0 +1,25 @@
+---
+'@remirror/react-core': minor
+---
+
+Add two new hook `useExtensionEvent` and `useExtensionCustomEvent`. You can use them to add event handlers to your extension. They are simpler and easier to use than the existed `useExtension` hook.
```

Comment:
It seems the only things that are using custom handlers, already have hooks that wrap them. i.e. `useKeymap` `useMultiPositioner` or `useSuggester`.

So I think we should avoid adding `useExtensionCustomEvent` to the public API (keep it for internal uses though). 

That way we avoid having to explain the difference between `handlers` and `customHandlers` in the documentation.

Reply:
OK. I will hide it

Reply:
done